pkg/operators/cli/clioperator.go

```diff
@@ -179,6 +179,45 @@ func (o *cliOperatorInstance) ExtraParams(gadgetCtx operators.GadgetContext) api
 	return api.Params{fields, mode}
 }
 
+func parseFields(fieldsString string, defaultFields []string) ([]string, error) {
+    fields := strings.Split(fieldsString, ",")
+    result := make([]string, len(defaultFields))
```

Comment:
What about using two maps, one for "hide" and one for "show" fields to be able to check things like the same field is hidden but shown, etc. 

Something similar to the implementation in https://github.com/inspektor-gadget/inspektor-gadget/blob/7ec66979122fb7e78bc34bd7cedf2e3db8eed24d/pkg/datasource/formatters/json/json.go#L94-L113. 

(One advantage of using map is that removing elements from it is very easy, also we don't need to worry about duplicates)

Reply:
Hi @mauriciovasquezbernal now using, separate maps for removing and adding fields. And also checking duplicates at any stage. And finally show combined result. Please review and provide your feedback

Reply:
Unfortunately I think the maps wasn't a good idea as it messes up with the order of the fields. I think we need to switch back to slices and use helpers like slices.Contains to avoid doing a lot of manual work.